### DISCUSSION TITLE
Keyboard shortcuts for adding fields/tags?

### DISCUSSION THREAD

**Discussion by SpoonOfDoom** [TagStudioDev/TagStudio#558] [Q&A]
Hi,

so I understand that keyboard navigation is an ongoing issue, but I cannot find *any* hotkey for adding fields or tags to an image without the mouse. Am I missing something, or do such shortcuts not exist yet? Can't really find anything about shortcuts in the documentation.

**Comment by CentaurWorks** [comment]
Please add something like this. I want to add more functionality to my Stream Deck(s) but I currently can't with the lack of keyboard shortcuts in general
